File: server/order-service/src/main/java/com/sudo248/orderservice/controller/order/dto/OrderProductInfoDto.java
package com.sudo248.orderservice.controller.order.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OrderProductInfoDto {
    private String productId;
    private String supplierId;
    private String sku;
    private String name;
    private Float price;
    private String brand;
    private List<String> images;
    private float weight;
    private float height;
    private float length;
    private float width;
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/dto/order/OrderProductInfoDto.kt
package com.sudo248.sudoo.data.dto.order

data class OrderProductInfoDto (
    val productId: String,
    val supplierId: String,
    val name: String,
    val sku: String,
    val images: List<String>,
    val price: Double,
    val brand: String,
    val weight: Float,
    val height: Float,
    val length: Float,
    val width: Float,
)